doc/whats_new.rst

```diff
@@ -20,3 +20,5282 @@ Previous Releases
     Version 0.14 <whats_new/v0.14.rst>
```

Comment:
This file was not correctly merged. Make sure this file is not touched in the diff, only `doc/whats_new/v0.20.rst`